host/cli/signal.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/pkg/cluster"
+)
+
+func init() {
+	Register("signal", runSignal, `
+usage: flynn-host signal ID [SIGNAL]
```

Comment:
Ahh yep, I was going to make it default to SIGINT but figured that probably doesn't make sense and didn't fully roll it back. Will fix it up.